packages/client/src/create-deployment.ts

```diff
@@ -50,8 +50,6 @@ export default function buildCreateDeployment() {
     clientOptions.isDirectory =
       !Array.isArray(path) && lstatSync(path).isDirectory();
 
-    let rootFiles: string[];
```

Comment:
This was just not being used?

Reply:
Yep, it was just used for that one log which was useless.

The root files were read again in `let { fileList } = await buildFileTree()` just below